tests/test_4_immut.py

```diff
@@ -126,6 +126,14 @@ def test_components_vasculature():
     assert_equal(len(morphology.section_types), 3080)
     assert_equal(len(morphology.section(0).predecessors), 0)
     assert_equal(len(morphology.section(0).successors), 2)
+
```

Comment:
This test function is becoming to big.
I think it's time to make a test_vasculature.py and create smaller test function there.

Reply:
I agree. So I will move all the vasculature tests to the new file.